rel/start.eex

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+set -e
+
+export RELEASE_COOKIE=$COOKIE
```

Comment:
we'll most likely end up replacing this file with some env.sh file, see: https://github.com/elixir-lang/elixir/pull/8988